vue.config.js

```diff
@@ -39,4 +39,24 @@ module.exports = {
       },
     },
   },
+
+  // In order for the sails cookie to be set on the client when browsing in dev mode, we need
+  // to proxy requests to the server
+  // https://webpack.js.org/configuration/dev-server/
+  devServer: {
+    proxy: {
+      '/game': {
+        target: 'http://localhost:1337',
+        changeOrigin: true,
+      },
+      '/user': {
+        target: 'http://localhost:1337',
+        changeOrigin: true,
+      },
+      '/test': {
+        target: 'http://localhost:1337',
+        changeOrigin: true,
+      },
+    },
+  },
```

Comment:
Need this for the cookie logic to work when running discrete client/server apps.